simple/stats/data.py

```diff
@@ -234,25 +235,45 @@ class Observation:
   date: str
   value: str
   provenance: str
+  unit: str = ""
+  scaling_factor: str = ""
+  measurement_method: str = ""
+  observation_period: str = ""
   properties: dict[str, str] = field(default_factory=dict)
 
   def __post_init__(self):
     if not self.properties:
       self.properties = {}
-    # Properties in the DB are stored as gzipped and base64 encoded strings.
-    # Convert it to json / dict so it is available as a dict in code.
+    # Properties in the DB are stored as stringified json.
+    # Load it as a dict in code.
```

Comment:
I feel this type conversion is too subtle, should the json.loads happen outside the class object constructor?

Reply:
Simplified this class and added a separate one explicitly for `ObservationProperties`.